samples/snippets/src/test/java/com/example/storage/ITHmacSnippets.java

```diff
@@ -49,7 +49,7 @@ public void before() {
     cleanUpHmacKeys(ServiceAccount.of(HMAC_KEY_TEST_SERVICE_ACCOUNT));
   }
 
-  @Rule public MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(5);
+  @Rule public MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(5, 2000L);
```

Comment:
Any specific reason why this needs a longer initial timeout compared to the others?

Reply:
Failed consistently 5 times (https://source.cloud.google.com/results/invocations/1b7ca05f-7f48-44d3-9a99-ebeb4fda0cec/targets/github%2Fjava-storage%2Fsamples%2Fsnapshot%2Ftarget%2Ffailsafe-reports%2Fsamples%2Fcom.example.storage.ITHmacSnippets/tests); considered increasing the initial retry to give the API time on the first attempt only;

IIUC, @ Before will be called before each attempt and clean up resources. The issue with this multiple executions of integration tests and this clean up step doesn't consider age of HMAC resources created.

Reply:
Fair enough. Yes, `@Before` will be called each attempt.